evap/staff/views.py

```diff
@@ -2120,6 +2121,52 @@ class ProgramIndexView(SuccessMessageMixin, SaveValidFormMixin, FormsetView):
     success_message = gettext_lazy("Successfully updated the programs.")
 
 
+@manager_required
+def program_merge_selection(request):
+    form = ProgramMergeSelectionForm(request.POST or None)
+
+    if form.is_valid():
+        main_instance = form.cleaned_data["main_instance"]
+        other_instance = form.cleaned_data["other_instance"]
+        return redirect("staff:program_merge", main_instance.id, other_instance.id)
+
+    return render(request, "staff_program_merge_selection.html", {"form": form})
+
+
+@manager_required
+def program_merge(request, main_id, other_id):
+    main_instance = get_object_or_404(Program, id=main_id)
+    other_instance = get_object_or_404(Program, id=other_id)
+
+    if request.method == "POST":
+        with transaction.atomic():
+            main_instance.import_names += other_instance.import_names
```

Comment:
This could lead to one name occurring multiple times in the list - we should introduce a database constraint eventually anyways